pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -35,18 +40,104 @@ import (
 	_ "github.com/docker/distribution/registry/storage/driver/s3-aws"
 	_ "github.com/docker/distribution/registry/storage/driver/swift"
 
-	"strings"
+	kubeversion "k8s.io/kubernetes/pkg/version"
 
 	"github.com/openshift/origin/pkg/cmd/server/crypto"
 	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
 	"github.com/openshift/origin/pkg/dockerregistry/server"
 	"github.com/openshift/origin/pkg/dockerregistry/server/api"
 	"github.com/openshift/origin/pkg/dockerregistry/server/audit"
 	registryconfig "github.com/openshift/origin/pkg/dockerregistry/server/configuration"
+	"github.com/openshift/origin/pkg/dockerregistry/server/prune"
+	"github.com/openshift/origin/pkg/version"
 )
 
+var pruneMode = flag.String("prune", "", "prune blobs from the storage and exit (check, delete)")
```

Comment:
I see. So in order to reduce the possibility for typo, there would have to be bool options like:

```
dockerregistry -prune -confirm    # preferred
# or
dockerregistry -prune -dry-run    # more consistent with oadm prune images
```

But the single dash option is nonstandard, so the user needs to see help message anyway or act upon doc instructions (do a copy&paste). So this is a non-issue.